File: app/src/main/java/za/co/lukestonehm/logicaldefence/About.java
package za.co.lukestonehm.logicaldefence;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;

public class About extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);
        Toolbar mActionBar = findViewById(R.id.action_bar);
        setSupportActionBar(mActionBar);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
    }
}

File: app/src/main/java/za/co/lukestonehm/logicaldefence/About.kt
package za.co.lukestonehm.logicaldefence

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar

class About : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about)
        val mActionBar = findViewById<View>(R.id.action_bar) as Toolbar
        setSupportActionBar(mActionBar)
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
    }
}